docs/guides/key-manager/give-permissions.md

```diff
@@ -128,43 +128,77 @@ The private key can be obtained depending on how you created your Universal Prof
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { ERC725 } from '@erc725/erc725.js';
+import LSP6Schema from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
 import Web3 from 'web3';
+
 const web3 = new Web3('https://rpc.l16.lukso.network');
+const myUniversalProfileAddress = '0x...';
 
 const PRIVATE_KEY = '0x...'; // your EOA private key (previously created)
-
 const myEOA = web3.eth.accounts.wallet.add(PRIVATE_KEY);
 ```
 
-### 3.2 - Create contract instance
+  </TabItem>
 
-The next steps is to create the web3 instance of our smart contracts to interact with them. The contract ABIs are available in the @lukso/lsp-smart-contracts npm package.
+  <TabItem value="ethersjs" label="ethers.js">
 
-You will need the address of your Universal Profile deployed on L16.
+```javascript title="Load account from a private key"
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { ERC725 } from '@erc725/erc725.js';
+import LSP6Schema from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
+import { ethers } from 'ethers';
 
-```js
-import UniversalProfile from "@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json";
-import KeyManager from "@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json";
-import Web3 from "web3";
+const myUniversalProfileAddress = '0x...';
+
+const PRIVATE_KEY = '0x...'; // your EOA private key (previously created)
+const provider = new ethers.JsonRpcProvider('https://rpc.l16.lukso.network');
+const myEOA = new ethers.Wallet(privateKey).connect(provider);
+```
+
+  </TabItem>
+
+</Tabs>
+
+### 3.2 - Create contract instance
 
-const RPC_ENDPOINT = "https://rpc.l16.lukso.network";
-const web3 = new Web3(RPC_ENDPOINT);
+The next steps is to create an instance of our UP smart contract to interact with. The contract ABIs are available in the @lukso/lsp-smart-contracts npm package.
 
+You will need the address of your Universal Profile.
 
-// step 1 - create instance of UniversalProfile and KeyManager contracts
-const myUniversalProfileAddress = "0x4F81bFDD12c73c94222C7879C91c1B837b8adb62";
-const myUniversalProfile = new web3.eth.Contract(UniversalProfile.abi, myUniversalProfileAddress);
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```js
+// step 1 - create instance of UniversalProfile contract
+const myUniversalProfile = new web3.eth.Contract(
+  UniversalProfile.abi,
+  myUniversalProfileAddress,
+);
 ```
 
-Since the KeyManager is the owner of the Universal Profile, its address can be obtained easily by querying the `owner()` of the Universal Profile.
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
 
 ```js
-const keyManagerAddress = await myUniversalProfile.methods.owner().call();
-const myKeyManager = new web3.eth.Contract(KeyManager.abi, keyManagerAddress);
+// step 1 - create instance of UniversalProfile contract
+const unviersalProfile = new ethers.Contract(
```

Comment:
typo

```suggestion
const universalProfile = new ethers.Contract(
```

Reply:
We also need to add the previous imports here, otherwise people don't know where the ABI comes from. See screenshot below.

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/31145285/232020962-f07f857b-978f-4709-8599-eca9155eed4c.png">